common/models/change.js

```diff
@@ -630,6 +637,74 @@ module.exports = function(Change) {
   };
 
   /**
+   * Resolve the conflict using the instance data in the source model.
+   *
+   * @callback {Function} callback
+   * @param {Error} err
+   */
+  Conflict.prototype.resolveUsingSource = function(cb) {
+    this.resolve(function(err) {
+      // don't forward any cb arguments from resolve()
+      cb(err);
+    });
+  };
+
+  /**
+   * Resolve the conflict using the instance data in the target model.
+   *
+   * @callback {Function} callback
+   * @param {Error} err
+   */
+  Conflict.prototype.resolveUsingTarget = function(cb) {
+    var conflict = this;
+
+    conflict.models(function(err, source, target) {
+      if (err) return done(err);
+      if (target === null) {
+        return conflict.SourceModel.deleteById(conflict.modelId, done);
+      }
+      var inst = new conflict.SourceModel(target);
```

Comment:
I cringe when I see this (creating an instance just to call `save()`). I'm guessing you are depending on something like auto-setting the ID properly. Otherwise cant you use `SourceModel.update()`?